README.md

```diff
@@ -154,22 +153,22 @@ may want to try to contact members of the community for help. The community is
 also where you want to go for questions about how to extend rebar, fill in bug
 reports, and so on.
 
-The main place to go for questions is the [rebar mailing
```

Comment:
Not sure there's an alternative to this mailing list.

Reply:
we're still on there but nobody writes there. We could likely extract this section and refer to https://www.rebar3.org/docs/about/about-us/#community

Reply:
I get a 404 for http://lists.basho.com/pipermail/rebar_lists.basho.com/ and a certificate error for https://lists.basho.com/pipermail/rebar_lists.basho.com/.

Reply:
You mean https://rebar3.org/docs/about/about-us/#community, right? (without the www.) 😄

Reply:
34af45c